cmd/controller/api/apiHandlers/authenticationHandlers.go

```diff
@@ -33,14 +35,31 @@ type Credentials struct {
 	NewPassword string `json:"newPassword"`
 }
 
-var cookieName = "purser-session-token"
+// User structure
+type User struct {
+	Username string
+	Authenticated bool
+}
+
+var cookieName = "purser-token-new"
```

Comment:
Are we using this anywhere?

Reply:
Yes, we need it to store and get the session. A cookiestore can support more than one type of cookie (cookiename)
`session, err := store.Get(r, cookieName)`
In the cookie key-value that backend provides to the UI, cookieName is the key and value is the encryption of the User object. This PR makes this encryption key random.